scripts/build_requirements.py

```diff
@@ -4,11 +4,11 @@
 
 from utils import Downloadable
 
-POOL_URL = "https://deb.debian.org/debian/pool/main"
+POOL_URL = "https://sourceforge.net/projects"
 
 SWIG = Downloadable(
-    name          = "swig-3.0.12.tar.gz",
-    unpacked_name = "swig-3.0.12",
-    url  = POOL_URL + "/s/swig/swig_3.0.12.orig.tar.gz",
-    hash = "7cf9f447ae7ed1c51722efc45e7f14418d15d7a1e143ac9f09a668999f4fc94d"
+    name          = "swig-4.0.1.tag.gz",
```

Comment:
`swig-4.0.1.tag.gz` should be `swig-4.0.1.tar.gz`

Reply:
Good catch! I am pushing the fix for this. It does not really matter since it is a temporary file name, so, it does not really matter what this name is.

Reply:
Missed the part about stable 3.0.12. That version does not support python 3.8. The garbage collection API has changed in 3.8, so, 3x just does not compile anymore with 3.8.